gateway-discovery-cm/src/main/java/org/apache/knox/gateway/topology/discovery/cm/ClouderaManagerServiceDiscovery.java

```diff
@@ -63,6 +68,9 @@
   static final String DEFAULT_USER_ALIAS = "cm.discovery.user";
   static final String DEFAULT_PWD_ALIAS  = "cm.discovery.password";
 
+  private static final String GATEWAY_SERVICES_ACCESSOR_CLASS  = "org.apache.knox.gateway.GatewayServer";
```

Comment:
Use `GatewayServer.class...` to get the name instead of hardcoding the string.

Reply:
This module doesn't depend on gateway-server...I think the services would have to be passed down to avoid this

Reply:
Added a dependency on gateway-server to facilitate the proposed change.